languages/elixir/exercises/concept/anonymous-functions/.docs/introduction.md

```diff
@@ -33,5 +32,3 @@ variable = &(&1 + 1)
 variable.(1)
 # => 2
 ```
```

Comment:
Maybe add a little introduction to bitwise operations here too? Or just mentioning which module has functions for them?